vendor/github.com/jacobsa/gcloud/gcs/gcsfake/bucket.go

```diff
@@ -127,6 +129,7 @@ type bucket struct {
 	clock timeutil.Clock
 	name  string
 	mu    syncutil.InvariantMutex
+	storageClient *storage.Client
```

Comment:
Are we making the real storage client here?
Something like https://stackoverflow.com/questions/39814108/how-to-unit-test-google-cloud-storage-in-golang would help?

Reply:
yes I am creating a new one.